src/dd-droppable.ts

```diff
@@ -120,7 +120,7 @@ export class DDDroppable extends DDBaseImplement implements HTMLElementExtendOpt
     }
     this.triggerEvent('dropout', ev);
 
-    if (DDManager.dropElement === this) {
+    if (!external && DDManager.dropElement === this) {
       delete DDManager.dropElement;
```

Comment:
this should still happen. move to line 114.
rename the var fromParent and instead the next comment should be:
// if we're still over a parent droppable, send it an enter as we don't get one from leaving nested children if it's not already calling us
if (!fromParent) {
  let parentDrop: DDDroppable;
  let parent: DDElementHost = this.el.parentElement;
  ....